doc/howto/move_instances.md

```diff
@@ -54,11 +54,20 @@ For containers, there is limited support.
 Virtual machines can be moved to another server while they are running, thus without any downtime.
 
 To allow for live migration, you must enable support for stateful migration.
-To do so, ensure the following configuration:
+To do so, ensure the following configuration if your instance is not on a shared storage backend:
 
 * Set {config:option}`instance-migration:migration.stateful` to `true` on the instance.
 * Set [`size.state`](devices-disk) of the virtual machine's root disk device to at least the size of the virtual machine's {config:option}`instance-resource-limits:limits.memory` setting.
 
+If you create an instance on a shared storage backend like Ceph RBD, you don't need to set any parameters to enable the live migration. When an instance is created on a shared storage backend, {config:option}`instance-migration:migration.stateful` is set to `true` by default for the instance. Additionally, the [`size.state`](devices-disk) parameter is unset, because it is not required for shared storage.
+
+The default value for {config:option}`instance-migration:migration.stateful` was introduced with LXD 5.21 and applies only to new instances. Existing instances won't be affected by this change.
```

Comment:
We dont tend to mention versions in the docs, but rather leave that to the API extension functionality.